masterlist.yaml

```
@@ -19568,6 +19568,13 @@ plugins:
       # version: 1.6
       - crc: 0xEAEAF38B
         util: 'SSEEdit v4.0.2f'
+  - name: 'Undriel_QRR_Clockwork_Patch.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/20851/' ]
+    dirty:
+      - <<: *reqManualFix
+        crc: 0xF3661E6A
```

I think you might've edited your copy of this plugin at some point, the checksum I get from a fresh install is different.

```suggestion
        crc: 0x8A98BC76
```

---

Yeah, it is marked as ESL in my load list...